From ad7dee95d244438814fbb7ed6d62a488cdff20f5 Mon Sep 17 00:00:00 2001
From: "Lu, Xingjiang" <xingjiang.lu@intel.com>
Date: Wed, 27 Mar 2024 18:25:47 +0800
Subject: [PATCH] [WA] Extended time to check for timeout

Extend check time to ensure Timecheck nto to send tombstone when
serviceManager was occupied and audiopolicyService cannot published
successfully.

Tracked-On: OAM-116189
Signed-off-by: Chen, Haochuan <haochuan.chen@intel.com>
Signed-off-by: Wan, Xinxin <xinxin.wan@intel.com>
Signed-off-by: Lu, Xingjiang <xingjiang.lu@intel.com>
---
 services/audiopolicy/service/AudioPolicyService.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/audiopolicy/service/AudioPolicyService.cpp b/services/audiopolicy/service/AudioPolicyService.cpp
index 20e305e864..9af00fd73f 100644
--- a/services/audiopolicy/service/AudioPolicyService.cpp
+++ b/services/audiopolicy/service/AudioPolicyService.cpp
@@ -1094,7 +1094,7 @@ status_t AudioPolicyService::onTransact(
     }
 
     std::string tag("IAudioPolicyService command " + std::to_string(code));
-    TimeCheck check(tag.c_str());
+    TimeCheck check(tag.c_str(), 12000);
 
     switch (code) {
         case SHELL_COMMAND_TRANSACTION: {
-- 
2.34.1

